invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentPendingLoans/DocumentPendingLoans.js

```diff
@@ -47,16 +47,16 @@ export default class DocumentPendingLoans extends Component {
     });
   }
 
-  _render_table(data) {
+  render_table(data) {
```

Comment:
Same here :)
```suggestion
  renderTable(data) {
```
I saw a few more `render_table` so would be nice if you could check those too. :)